core/src/banking_stage.rs

```diff
@@ -427,20 +413,11 @@ impl BankingStage {
                 batch_limit,
                 transaction_status_sender.clone(),
                 &gossip_vote_sender,
+                &mut buffered_packets,
             ) {
                 Err(RecvTimeoutError::Timeout) => (),
                 Err(RecvTimeoutError::Disconnected) => break,
-                Ok(mut unprocessed_packets) => {
-                    if unprocessed_packets.is_empty() {
-                        continue;
-                    }
-                    let num: usize = unprocessed_packets
-                        .iter()
-                        .map(|(_, unprocessed)| unprocessed.len())
-                        .sum();
-                    inc_new_counter_info!("banking_stage-buffered_packets", num);
-                    buffered_packets.append(&mut unprocessed_packets);
-                }
+                _ => (),
```

Comment:
As this could swallow any newly-added `Err`s (maintenance burden), I think this should be `Ok(())`.

Reply:
or more like this:

```rust
  Ok(()) | Err(RecvTimeoutError::Timeout) => (),
  Err(RecvTimeoutError::Disconnected) => break,
```

Reply:
agreed, changed!